test/unit/modules/accounts.js

```diff
@@ -84,6 +84,7 @@ describe('account', function () {
 			expect(accounts.generateAddressByPublicKey(validAccount.publicKey)).to.equal(validAccount.address);
 		});
 
+		// ToDo: Design a throwable test
 		it.skip('should throw error for invalid publicKey', function () {
```

Comment:
When is this gonna be implemented?

Reply:
This issue is there due to the way we currently use `Buffer.from`, I think that is out of scope for the current PR